python/ray/tests/test_basic_3.py

```diff
@@ -433,5 +433,14 @@ def run():
     ray.get(verify_job_id.remote(job_id, True))
 
 
+def test_kv_multi_get(ray_start_regular):
```

Comment:
Could we split the multi-get changes into a separate PR?

Reply:
If you wish, but I'd like to have a separate PR for other changes and convert this one for multi-get only.